framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/ProjectBuilding/Analyticses/CliAnalyticsCollect.cs

```diff
@@ -32,27 +33,37 @@ public class CliAnalyticsCollect : ICliAnalyticsCollect, ITransientDependency
         public async Task CollectAsync(CliAnalyticsCollectInputDto input)
         {
             var postData = _jsonSerializer.Serialize(input);
-            using (var client = new CliHttpClient())
+            var url = $"{CliUrls.WwwAbpIo}api/clianalytics/collect";
+            
+            try
             {
-                var responseMessage = await client.PostAsync(
-                    $"{CliUrls.WwwAbpIo}api/clianalytics/collect",
-                    new StringContent(postData, Encoding.UTF8, MimeTypes.Application.Json),
-                    _cancellationTokenProvider.Token
-                );
-
-                if (!responseMessage.IsSuccessStatusCode)
+                using (var client = new CliHttpClient())
                 {
-                    var exceptionMessage = "Remote server returns '" + (int)responseMessage.StatusCode + "-" + responseMessage.ReasonPhrase + "'. ";
-                    var remoteServiceErrorMessage = await _remoteServiceExceptionHandler.GetAbpRemoteServiceErrorAsync(responseMessage);
+                    var responseMessage = await client.PostAsync(
+                        url,
+                        new StringContent(postData, Encoding.UTF8, MimeTypes.Application.Json),
+                        _cancellationTokenProvider.Token
+                    );
 
-                    if (remoteServiceErrorMessage != null)
+                    if (!responseMessage.IsSuccessStatusCode)
                     {
-                        exceptionMessage += remoteServiceErrorMessage;
-                    }
+                        var exceptionMessage = "Remote server returns '" + (int)responseMessage.StatusCode + "-" + responseMessage.ReasonPhrase + "'. ";
+                        var remoteServiceErrorMessage = await _remoteServiceExceptionHandler.GetAbpRemoteServiceErrorAsync(responseMessage);
+
+                        if (remoteServiceErrorMessage != null)
+                        {
+                            exceptionMessage += remoteServiceErrorMessage;
+                        }
 
-                    _logger.LogInformation(exceptionMessage);
+                        _logger.LogInformation(exceptionMessage);
+                    }
                 }
             }
+            catch (Exception ex)
+            {
+                Console.WriteLine("Error occured while cli analytics from {0} : {1}", url, ex.Message);
+            }
+            
```

Comment:
The exception message of the collect method should not be output to the developer.